### ISSUE TITLE
Incorrect BoundingRectangle value

### ISSUE THREAD

**Issue by ThinhVu** [yinkaisheng/Python-UIAutomation-for-Windows#129]
Hi, 
Thank you for the library.

I trying to capture the "busy" control in this application but the problem is BoundingRectangle property of this control is incorrect.

Window size of this application is smaller than it's actual size which shown in Inspect.exe too.

Can you explain what is the root cause of this issue and how to fix?

<img width="1004" alt="Capture" src="https://user-images.githubusercontent.com/8122962/89297668-72538e00-d68e-11ea-8564-c8e37438db03.PNG">

**Comment by BruceLee569** [comment]
+1

I think the value returned by `BoundingRectangle` is incorrect because the system has set `the display zoom ratio`. Multiplying `BoundingRectangle` by the zoom ratio on my computer should be the correct value.

@yinkaisheng  I hope the author can fix this problem. There is another problem: I got the address bar of the Chrome browser, which is an EditControl, but how do I get the url text in it? I tried `GetWindowText`, it returned None, and `SetWindowText` did not take effect.
